src/redux/actions/cellSets/createCellSetByExpression.js

```diff
@@ -10,14 +8,8 @@ const createCellSetByExpression = (
     genesConfig: selectedGenes,
   };
 
-  try {
-    const result = await fetchWork(experimentId, body, getState, { broadcast: true });
-    return result;
-  } catch (e) {
-    console.error(e);
-    pushNotificationMessage('error', endUserMessages.ERROR_FETCHING_CELL_SETS);
-    throw new Error(e);
-  }
```

Comment:
why are you removing all the error handling in here?

Reply:
This error moved to be logged and handled in the `experimentUpdatesHandler`. We can let the error bubble up to the level above to be handled. If we handle both here and in `experimentUpdatesHandler`, we'll see double error. If we handle it here but not in `experimentUpdatesHandler`, only the user running the createCellSet will see the If we handle it in `experimentUpdatesHandler`, all users currently opening the experiment will see the error.

Reply:
I don't see the reason why all users should receive a notification for failed cluster creation action. Other users won't know that someone is trying to create a cell set. Getting a failed notification will only confuse them and make them think that they did something that caused the action.

Reply:
I think we need a proper discussion about this issue, i.e. how we should notify multiple concurrent users viewing the experiment, to be clear on our stance.

Reply:
Iva has clarified about this during the stand up. Only the user dispatching the work request should be notified, there is no need for others to know about the failed cluster creation. PR has been updated to reflect this.